File: Nanodegree/CheckProfanity/check_profanity_rb/check_profanity.rb
require 'net/http'
require 'json'


def read_text()
  movie_quotes = File.read('/Volumes/Macintosh HD/Users/SilverMaiden/Downloads/movie_quotes.txt')
  check_profanity(movie_quotes)

end

def check_profanity(text_to_check)
  query = "select *"
  url = "http://www.purgomalum.com/service/containsprofanity?text=" + text_to_check
  response = Net::HTTP.get_response(URI.parse(url))
  output = response.body
  if output == "true"
    puts "Profanity Alert!!!"
  elsif output == "false"
    puts "No Profanity Detected."
  else
    puts "Could Not Properly Check Document."
  end

end

read_text

File: Nanodegree/CheckProfanity/check_profanity_py/check_profanity.py
import urllib


def read_text():
    movie_quotes = open("/Volumes/Macintosh HD/Users/SilverMaiden/Downloads/movie_quotes.txt")
    contents_of_file = movie_quotes.read()
    movie_quotes.close()
    check_profanity(contents_of_file)
    #print(contents_of_file)

def check_profanity(text_to_check):
    connection = urllib.urlopen("http://www.purgomalum.com/service/containsprofanity?text=" + text_to_check)
    output = connection.read()
    if output == "true":
        print("Profanity Alert!!")
    elif output == "false":
        print("No Profanity Detected.")
    else:
        print("Could Not Scan Document Properly.")

   # print(output)
    connection.close()

read_text()